doc/client_lua_api.txt

```diff
@@ -30,6 +30,16 @@ Startup
 Mods are loaded during client startup from the mod load paths by running
 the `init.lua` scripts in a shared environment.
 
+In order to load client-side mods in your worlds, you have 2 conditions :
+
+ 1) $path_user/minetest.conf contains the setting `enable_client_modding = true`
+
+ 2) The client-side mod located in $path_user/clientmods/<modname> is added to
+    $path_user/clientmods/mods.conf as `load_mod_<modname> = true` in order to load*.
```

Comment:
The final `in order to load` is redundant, due to `In order to load client-side mods in your worlds, you have 2 conditions :`